docs/website/docs/overview/installation.md

```diff
@@ -31,12 +31,12 @@ Installing `odo` on `amd64` architecture:
 
 1. Download the latest release from the mirror:
 ```shell
-curl -L https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/v3.0.0~alpha2/odo-linux-amd64 -o odo
+curl -L https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odov3.0.0~beta1odo-linux-amd64 -o odo
```

Comment:
Missing `/` between `~beta1` and `odo-linux` here and below ;-)

Reply:
Ah, and also between `odo` and `v3.0.0~beta1`

Reply:
:facepalm: Fixed it, thanks for catching!